controllers/controllers.js

```diff
@@ -66,3 +67,10 @@ exports.getUsers = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  deleteCommentById(comment_id).then((result) => {
+    res.status(204).send({ deleted: result });
```

Comment:
up to you but you can probably get rid of the send here, I suspect it's not doing anything for us and 204 is enough for showing that the delete request has been successful